bindings/python/src/session.cpp

```diff
@@ -388,6 +388,22 @@ namespace
     }
 #endif // TORRENT_ABI_VERSION
 
+    object alert_notify_cb;
```

Comment:
if you have more than one session instance, this won't work. Instead, bind the python `object` into the function object. Like, `s.set_alert_notify(std::bind(&alert_notify, cb));`

and make `alert_notify()` take the `object` as a parameter.